src/main/java/seedu/address/logic/parser/CliSyntax.java

```diff
@@ -16,4 +16,5 @@ public class CliSyntax {
     public static final Prefix PREFIX_UNITNUMBER = new Prefix("unitNo/");
     public static final Prefix PREFIX_BLOCK = new Prefix("blk/");
     public static final Prefix PREFIX_POSTALCODE = new Prefix("postal/");
+    public static final Prefix PREFIX_BUDGET = new Prefix("b/");
```

Comment:
Do you think it is better to explicitly name the prefix as budget/ ?

Not really a big issue, just a personal preference.

Reply:
Should be fine I feel, like name is also "n/". We can discuss during meeting.